tests/testthat/test-CRANSKIP-logger-namespaces.R

```diff
@@ -1,95 +1,28 @@
+      quiet = TRUE
+    )
+    withr::defer(devtools::unload('logger.tester'))
+  
+    local_test_logger(layout = layout_glue_generator("{ns} {level} {msg}"))
+    expect_snapshot({
+      logger_tester_function(INFO, 'x = ')
+      logger_info_tester_function('everything = ')
+    })
 })
 
-rscript <- function(path) {
-  out <- system(paste('$R_HOME/bin/Rscript', path, '2>&1'), intern = TRUE)
-  setdiff(out, "NULL")
-}
-
-test_that('namespace in a remote R session to avoid calling from testthat', {
-
-    t <- tempfile()
-    cat('
-      library(logger)
-      log_layout(layout_glue_generator("{ns} / {ans} / {topenv} / {fn} / {call}"))
-      log_info("foobar")', file = t)
-    expect_equal(
-        rscript(t),
-        'global / global / R_GlobalEnv / NA / NA')
-    unlink(t)
-
-    t <- tempfile()
-    cat('
-      library(logger)
-      log_layout(layout_glue_generator("{ns} / {ans} / {topenv} / {fn} / {call}"))
-      f <- function() log_info("foobar")
-      f()', file = t)
-    expect_equal(
-        rscript(t),
-        'global / global / R_GlobalEnv / f / f()')
-    unlink(t)
-
-    t <- tempfile()
-    cat('
-      library(logger)
-      log_layout(layout_glue_generator("{ns} / {ans} / {topenv} / {fn} / {call}"))
-      f <- function() log_info("foobar")
-      g <- function() f()
-      g()', file = t)
-    expect_equal(
-        rscript(t),
-        'global / global / R_GlobalEnv / f / f()')
-    unlink(t)
-
-    t <- tempfile()
-    cat('
-      library(logger)
-      log_layout(layout_glue_generator("{ns} / {ans} / {topenv} / {fn} / {call}"))
-      f <- function() log_info("foobar")
-      g <- f
-      g()', file = t)
-    expect_equal(
-        rscript(t),
-        'global / global / R_GlobalEnv / g / g()')
-    unlink(t)
-
-    t <- tempfile()
-    cat('
-      library(logger)
-      log_layout(layout_glue_generator("{ns} / {ans} / {topenv} / {fn} / {call}"))
-      log_appender(appender_stdout)
-      devtools::load_all(system.file("demo-packages/logger-tester-package", package = "logger"), quiet = TRUE)
-      logger_info_tester_function("foobar")',
-      file = t)
-    expect_equal(
-        rscript(t),
-        'logger.tester / global / logger.tester / logger_info_tester_function / logger_info_tester_function("foobar")')
-    unlink(t)
-
-    t <- tempfile()
-    cat('
-      library(logger)
-      log_layout(layout_glue_generator("{ns} / {ans} / {topenv} / {fn} / {call}"))
-      devtools::load_all(system.file("demo-packages/logger-tester-package", package = "logger"), quiet = TRUE)
-      log_threshold(INFO, namespace = "logger.tester")
-      log_appender(appender_stdout, namespace = "logger.tester")
-      logger_info_tester_function("foobar")',
-      file = t)
-    expect_equal(
-        rscript(t),
-        'logger.tester / logger.tester / logger.tester / logger_info_tester_function / logger_info_tester_function("foobar")')
-    unlink(t)
+test_that('log_info() captures local info', {
+    local_test_logger(
```

Comment:
Again, not sure why this was running in separate process, but the snapshot outputs from the active process look fine to me.

Reply:
it's been a long time ago :zipper_mouth_face: but if I recall correctly, I had some issues with `testthat` injecting frames in the call stack ... but happy to hear it's working :tada: